chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/schedule/ChplOneTimeTrigger.java

```diff
@@ -0,0 +1,27 @@
+package gov.healthit.chpl.domain.schedule;
+
+import java.io.Serializable;
+
+public class ChplOneTimeTrigger implements Serializable {
```

Comment:
Why doesn't this class need a name or group like the ChplTrigger class has? Purely based on the class names, it looks like it should be more of a hierarchy with parent class ChplTrigger and subclasses RepeatableChplTrigger and OneTimeChplTrigger or something like that. Would any refactoring like that be appropriate here or is it not really the right thing to do? I'm not as familiar with the objects as you are.